README.md

```diff
@@ -284,21 +277,31 @@ will return:
         "aim": {aim},
         "expectationsForMembers": [],
```

Comment:
Sure thing - I am not sure what it is, and I cant really see it from the mocks. :) What should the initial value be?